pkg/backend/filestate/backend.go

```diff
@@ -291,7 +291,7 @@ func (b *localBackend) apply(ctx context.Context, kind apitype.UpdateKind, stack
 	// Create a separate event channel for engine events that we'll pipe to both listening streams.
 	engineEvents := make(chan engine.Event)
 
-	scope := op.Scopes.NewScope(engineEvents, dryRun)
+	scope := op.Scopes.NewScope(events, dryRun)
```

Comment:
Assuming this the right channel to be passing here - should we move `engineEvents` below these lines to make it clearer what it is logically used for?

Reply:
Sure, if that helps when visualizing the object lifetimes here.